### PR TITLE
Update play mailer to 6.0.1

### LINKED ISSUE #56: Update to play-mailer 6.0.1
Update to play-mailer 6.0.1 which updates commons-email to v1.5 which fixes CVE-2017-9801 : https://lists.apache.org/thread.html/7ef903a772a2ff08605df1be819044fb15df2815eb3d63878b3fbbb5@%3Cannounce.apache.org%3E

### PR DESCRIPTION
**Pull request by albuch**
Fixes #56

### CODE REVIEW

File: code/build.sbt [hunk 1]
```diff
@@ -2,11 +2,12 @@ organization := "com.feth"
 
 name := "play-easymail"
 
+scalaVersion := "2.11.11"
```
Comment (joscha) [inline code/build.sbt]: was this needed @albuch or only added because of using sbt w/o `+`?

Comment (albuch) [inline code/build.sbt]: Intellij IDEA will not be able to compile properly without it. I'll file a bug there as it should probably pick up the first of `crossScalaVersions` if no explicit `scalaVersion` is set.

**Comment by joscha** [comment]
There should be a snapshot version released automatically from the master build - let me know if that version works for you, then I can do a release.

File: code/build.sbt (hunk 1 above)
Comment (albuch) [inline code/build.sbt]: For reference: https://youtrack.jetbrains.com/issue/SCL-12415

**Comment by albuch** [comment]
Thanks, I won't be able to test the snapshot soon though as we're still on play 2.5.x. If I find time on the weekend I'll check with a new project setup.

**Comment by albuch** [comment]
Hi @joscha,
I've tested the Snapshot version 0.9.4-SNAPSHOT today after upgrading to play 2.6 everything worked from our side. Could you cut a release at your earliest convenience?

**Comment by joscha** [comment]
just released on sonatype, should appear shortly.

**Pull request merged**